### PR TITLE
3.11 feature list

### PR DESCRIPTION
**Pull request by melcorr**
I know that at least one is missing. 
If anyone spots anything wrong or missing, please let me know.

### CODE REVIEW

File: _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### Worker timeout is no longer hardcoded
+
+In previous versions of Pulp, worker timeout was hardcoded as 30 seconds. Depending on the system, and the I/O of Postgresql, this hardcoded value caused problems.
+
+As part of this release, a new setting `WORKER_TTL` has been added. You can specify the interval, in seconds, that you consider a worker as missing after its last heartbeat. The default value is 30 seconds.
+
+You can edit the `WORKER_TTL` setting in `settings.py`. [#8291](https://pulp.plan.io/issues/8291)
+
+### Systems with existing Artifacts will undergo an automatic update as part of this release
```
Comment (daviddavis) [inline _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md]: I would probably move this section's text into the "Removed `md5` and `sha1` from ALLOWED_CONTENT_CHECKSUMS" section.

**Comment by daviddavis** [comment]
We added this in 3.11 too: https://pulp.plan.io/issues/7986

@pavelpicka added some good documentation about it in the PR: https://github.com/pulp/pulpcore/pull/1173/files#diff-1d78ba60e1efa4b7b08982e1bbaeceffefea079d402b7d58e57c0cfc7e6c78d8R274

**Comment by daviddavis** [comment]
Also it might be worth calling out the report is in tech preview.

**Comment by melcorr** [comment]
Thank you so much @daviddavis

File: _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+An error will now occur if you sync content that includes a checksum that is not part of `ALLOWED_CONTENT_CHECKSUMS`. [#7854](https://pulp.plan.io/issues/7854)
+
+### handle-artifact-checksums - now fully supported
+
+ The `handle-artifact-checksums`, command, which was introduced as a tech preview is now a fully-supported part of Pulp 3. [#7928](https://pulp.plan.io/issues/7928)
+
+### Update to `/pulp/api/v3/status/`
+
+In earlier versions of Pulp plugins, `/pulp/api/v3/status/` listed Python package names instead of Django app names. For plugins to have compatibility with Pulpcore 3.10, they needed to list Django app name and version.
```
Comment (mdellweg) [inline _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md]: This was a requirement for plugins compatible with 3.10, and we started to use that now. Still it sounds strange to tell something about 3.10 here. I'd remove that sentence or move it to the end of the paragraph.
Maybe rephrased like:
"To support this change, plugins are required to supply their label and version in the PulpPluginAppConfig since 3.10."

Comment (melcorr) [reply _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md]: Yessir :)

File: _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Let's take a look at the main features of this release.
+
+### Proxy Authorization Updates
+In previous versions of Pulp, the only way to specify a proxy username and password was to append them in the `proxy_url` fields.
+
+With this release, there are two new fields: `proxy_username` and `proxy_password` for remotes that you can use to enter your credentials. As a result of this change, you can no longer specify credentials as part of the `proxy_url` field.
+  A data migration will move the proxy auth information on existing remotes to the new fields. [#8167](https://pulp.plan.io/issues/8167)
+
+### Worker timeout is no longer hardcoded
```
Comment (daviddavis) [inline _posts/2021-03-15-pulpcore-3.11-is-now-generally-available.md]:
```suggestion
### Worker timeout is now configurable
```

**Comment by ipanova** [comment]
there are 2 more removals that should have happened in 3.10 but apparently we forgot https://pulp.plan.io/issues/8397 https://pulp.plan.io/issues/8398
